### PR TITLE
Completed Task 1.2

### PR DESCRIPTION
**Pull request by Lexanus686**
На проверочных данных я получил accuracy = 0.254 при learning_rate=1e-5 и reg_strength=1e-3 (если я правильно понял свой же output)
На тестовых данных accuracy=0.210

### CODE REVIEW

**Comment by Lexanus686** [comment]
Заранее извиняюсь за плохое знание git'а, но т.к. PR еще открыт, залью сюда Task 2.1
По итогу получил следующие результаты:
Loss: 35.633021, Train accuracy: 0.637444, val accuracy: 0.605000

**Comment by Lexanus686** [comment]
Лишние которые binary files в Task2.1?
Или .ipynb тоже удалять надо?
